README.md

```diff
@@ -1,2 +1,47 @@
-# freestyle-seed
-Freestyle g8 template 
+This is a Giter8 template for Freestyle projects.
+
+## Creating New Freestyle Project
```

Comment:
I'd update the steps in this file to be aligned what we are providing with this g8 template.